apps/root/src/services/meanApiService.ts

```diff
@@ -29,6 +29,11 @@ import { CLAIM_ABIS } from '@constants/campaigns';
 // MOCKS
 import { getProtocolToken, getWrappedProtocolToken } from '@common/mocks/tokens';
 import { Address, PublicClient, getContract } from 'viem';
+import { SavedCustomConfig } from '@state/base-types';
+
+type AccountWithConfig = Account & {
```

Comment:
We can move this to a types file

Reply:
Ish, it actually uses a type that is explicitely exclusive to the app, so we don't have access to RootState from the types package :(